### ISSUE TITLE
[Bug Report] New pre-commit setup seems to have broken black

### ISSUE THREAD

**Issue by RedTachyon** [openai/gym#2722]
tl;dr black is broken in the CI and in local pre-commit, at best failing style checks, at worst preventing commits from being made

---

So while updating #2712 I decided to finally setup the pre-commit thingie from #2602, and it seems that `black` is crashing there, preventing any actual commits. Then I realized that the same error is actually present in the CI, so it's not just a case of a messed up local installation. 

If you look at the [CI of the last commit there](https://github.com/openai/gym/runs/5778542997?check_suite_focus=true), there's a crash in `black` with the following traceback:

```
Traceback (most recent call last):
  File "/home/runner/.cache/pre-commit/repo9kskur0q/py_env-python3.10/bin/black", line 8, in <module>
    sys.exit(patched_main())
  File "/home/runner/.cache/pre-commit/repo9kskur0q/py_env-python3.10/lib/python3.10/site-packages/black/__init__.py", line 1423, in patched_main
    patch_click()
  File "/home/runner/.cache/pre-commit/repo9kskur0q/py_env-python3.10/lib/python3.10/site-packages/black/__init__.py", line 1409, in patch_click
    from click import _unicodefun
ImportError: cannot import name '_unicodefun' from 'click' (/home/runner/.cache/pre-commit/repo9kskur0q/py_env-python3.10/lib/python3.10/site-packages/click/__init__.py)
```

My best guess is that the current version of `black` defined for the CI is messed up, but I'm very much uncertain. If someone can figure it out that'd be great, otherwise I'll try to solve it within 5-7 business days.

**Comment by RedTachyon** [comment]
Pinging @kir0ul because maybe you have an idea what's going on

**Comment by kir0ul** [comment]
It seems to be a bug with Black that has been solved in version 22.3.0: https://github.com/psf/black/issues/2964#issuecomment-1080974737. Hopefully it should be solved by simply updating our version of Black in `.pre-commit-config.yaml` as below:

```diff
 ---
 repos:
   - repo: https://github.com/python/black
-    rev: 22.1.0
+    rev: 22.3.0
     hooks:
       - id: black
   - repo: https://github.com/codespell-project/codespell
```
